lib/cleanFiles.js

```diff
@@ -29,9 +29,33 @@ function cleanFiles(dictionary, platform) {
     throw new Error('Build path must end in a trailing slash or you will get weird file names.')
   }
 
-  // while neither the template, format, or dictionary are used by clean file I'm passing them in for symmetry to buildFile
+  var warningText = `
+    templates will be removed in Style Dictionary 3.0, please change your config to use formats.
+    This is an example of how to fix your config.js:
+
+    Before:
+      "files": [{
+        "destination": "font_dimens.xml",
+        "template": "android/fontDimens"
+      },{
+        "destination": "colors.xml",
+        "template": "android/colors"
+      }]
+
+    After:
+      "files": [{
+        "destination": "font_dimens.xml",
+        "format": "android/fontDimens"
+      },{
+        "destination": "colors.xml",
+        "format": "android/colors"
+      }]
+  `;
+
+  // while neither the format or dictionary are used by clean file I'm passing them in for symmetry to buildFile
   _.each(platform.files, function (file) {
     if (file.template) {
+      console.error(warningText);
       cleanFile(file.destination, file.template.bind(file), platform, dictionary);
```

Comment:
`file.format` ?

Reply:
I think this will be called if someone uses `files: [{template:'foo'}]` in their config. My goal was to ensure that they would see this warning but that the SD would still execute as it did previously.

@dbanksdesign could you double check this?